tests/apollo/test_skvbc_block_accumulation.py

```diff
@@ -0,0 +1,143 @@
+# Concord
+#
+# Copyright (c) 2020 VMware, Inc. All Rights Reserved.
+#
+# This product is licensed to you under the Apache 2.0 license (the "License").
+# You may not use this product except in compliance with the Apache 2.0 License.
+#
+# This product may include a number of subcomponents with separate copyright
+# notices and license terms. Your use of these subcomponents is subject to the
+# terms and conditions of the subcomponent's license, as noted in the LICENSE
+# file.
+
+import os.path
+import unittest
+import trio
+import random
+
+from util.bft import with_trio, with_bft_network, KEY_FILE_PREFIX, with_constant_load
+from util.skvbc_history_tracker import verify_linearizability
+from util import skvbc as kvbc
+
+import util.bft_network_partitioning as net
+import util.eliot_logging as log
+
+SKVBC_INIT_GRACE_TIME = 2
+BATCH_SIZE = 10
+NUM_OF_SEQ_WRITES = 1
+NUM_OF_PARALLEL_WRITES = 100
+MAX_CONCURRENCY = 10
+SHORT_REQ_TIMEOUT_MILLI = 3000
+LONG_REQ_TIMEOUT_MILLI = 15000
+BATCH_BY_REQ_NUM = "2"
+MAX_REQ_NUM_IN_BATCH = 5
+BATCH_FLUSH_PERIOD = "250"
+
+def start_replica_cmd(builddir, replica_id, view_change_timeout_milli="10000"):
+    """
+    Return a command that starts an skvbc replica when passed to
+    subprocess.Popen.
+
+    The replica is started with a short view change timeout.
+
+    Note each arguments is an element in a list.
+    """
+
+    status_timer_milli = "500"
+
+    path = os.path.join(builddir, "tests", "simpleKVBC", "TesterReplica", "skvbc_replica")
+    return [path,
+            "-k", KEY_FILE_PREFIX,
+            "-i", str(replica_id),
+            "-s", status_timer_milli,
+            "-v", view_change_timeout_milli,
+            "-b", BATCH_BY_REQ_NUM,
+            "-q", str(MAX_REQ_NUM_IN_BATCH),
+            "-z", BATCH_FLUSH_PERIOD,
+            "-u", str(True)
+            ]
+
+class SkvbcBlockAccumulationTest(unittest.TestCase):
+
+    __test__ = False  # so that PyTest ignores this test scenario
+
+    async def send_single_batch_write_with_pre_execution_and_kv(self, skvbc, client, batch_size, long_exec=False, ):
```

Comment:
Since this function is different from the one in test_skvbc_batch_preexecution.py maybe change the name, so it's clearer that it isn't just a copy of that.

Reply:
Updated code with changes